dat/missions/dvaered/dvaered-colonel-transport.lua

```diff
@@ -1,47 +1,92 @@
 --[[
 <?xml version='1.0' encoding='utf8'?>
-<mission name="Escort a Dvaered colonel to a system">
+<mission name="Dvaered Colonel Escort">
  <unique />
- <priority>1</priority>
- <chance>24</chance>
+ <priority>4</priority>
+ <chance>86</chance>
  <location>Bar</location>
  <spob>Dvaer Prime</spob>
 </mission>
 --]]
 --[[
 
-   MISSION: ESCORT TO A PLANET
-   DESCRIPTION: SMALL MISSION WHERE YOU ESCORT AN ARSENAL TO A PLANET: WIP
+   Mission: Escort a Dvaered colonel
+   Description: Small mission where you escort a Dvaered Arsenal
 
 --]]
 
 local fmt = require "format"
+local vn = require "vn"
+local vni = require "vnimage"
 
 
-function create ()
-   mem.talked = false
-   misn.setNPC( _("A Dvaered colonel"), "devaered/dv_military_f6.webp", _("This soldier seems to be a colonel.") )
+local npc_name = _("Dvaered Colonel")
+local npc_portrait, npc_image
+function create()
+   npc_image, npc_portrait = vni.dvaered
```

Comment:
You'll have to save the npc data with the mission or it will be lost on load. You can do this by storing it in the special `mem` table like:
```lua
mem.npc_image, mem.npc_portrait = vni.dvaeredMilitary()
```